docs/sources/enterprise/saml.md

```diff
@@ -204,6 +204,61 @@ org_mapping = Engineering:2, Sales:3
 allowed_organizations = Engineering, Sales
 ```
 
+## Set up SAML with Okta
+
+To configure SAML integration with Okta, create integration inside the Okta org first.
+
+1. Log in to the [Okta portal](https://login.okta.com/).
+1. Go to the Admin Console in your Okta org by clicking **Admin** in the upper-right corner. If you are in the Developer Console, click **Developer Console** in the upper-left corner and then click **Classic UI** to switch over to the Admin Console.
+1. In the Admin Console, go to **Applications** > **Applications**.
```

Comment:
By "go to" do you mean "click"? Navigate to? Please use precise language.

Reply:
"Navigate" is better I think.